source/Sashimi/Sashimi.csproj

```diff
@@ -26,7 +26,7 @@
   <ItemGroup>
     <PackageReference Include="Octopus.CoreParsers.Hcl" Version="1.1.2" />
     <PackageReference Include="Octopus.Dependencies.TerraformCLI" Version="1.0.9" />
-    <PackageReference Include="Sashimi.Server.Contracts" Version="7.1.0" />
+    <PackageReference Include="Sashimi.Server.Contracts" Version="8.5.4" />
```

Comment:
This is quite a big jump [diff here](https://github.com/OctopusDeploy/Sashimi/compare/7.1.0...8.5.4) I haven't looked at all of the implications, but are we happy with "If the tests are green, we are good to go" here?